hugegraph-dist/src/assembly/static/bin/start-hugegraph.sh

```diff
@@ -79,13 +79,20 @@ if [ ! -d "$LOGS" ]; then
     mkdir -p "$LOGS"
 fi
 
+GERMLIN_SERVER_YAML="gremlin-server.yaml"
+
+if [ $PRELOAD ]; then
```

Comment:
Some tiny suggestion, better use double square brackets to avoid exception if user don't set `PRELOAD`, also, we can add condition check whether the value is true. Like: `if [[ $PRELOAD = "true" ]]; then`.

Reply:
thx for your suggestion

Reply:
> Some tiny suggestion, better use double square brackets to avoid exception if user don't set `PRELOAD`, also, we can add condition check whether the value is true. Like: `if [[ $PRELOAD = "true" ]]; then`.

thx for your suggestion